pkg/kubelet/logs/container_log_manager.go

```diff
@@ -213,63 +223,93 @@ func (c *containerLogManager) Clean(ctx context.Context, containerID string) err
 	return nil
 }
 
+func (c *containerLogManager) processQueueItems(worker int) {
+	klog.V(4).InfoS("Starting container log rotate worker", "workerID", worker)
+	for c.processContainer(context.Background(), worker) {
+	}
+	klog.V(4).InfoS("Terminating container log rotate worker", "workerID", worker)
+}
+
 func (c *containerLogManager) rotateLogs(ctx context.Context) error {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
+	klog.V(4).InfoS("Starting container log rotation sequence")
 	// TODO(#59998): Use kubelet pod cache.
 	containers, err := c.runtimeService.ListContainers(ctx, &runtimeapi.ContainerFilter{})
```

Comment:
Lots of workers each running `c.runtimeService.ListContainers` is gonna be costly especially in a big node with LOTS of containers :(